env_files/fidesctl.env

```diff
@@ -1 +1,2 @@
 FIDESCTL_CONFIG_PATH="/fides/fidesctl/fidesctl.toml"
+FIDESCTL_TEST_MODE="True"
```

Comment:
@ThomasLaPiana  What's going on here? Is this always True now?

Reply:
yep, the make commands/docker-compose in the repo is only meant for development purposes, so its safe to always have this on here

this PR will prevent random people from accidentally running the tests locally and destroying their prod data

Reply:
Ah, I see. Wasn't really thinking of it that way but it does make sense.